src/pages/guides/using/asset-events/asset-events-actions.md

```diff
@@ -4,19 +4,19 @@
 
```

Comment:
There is an extra line break (`<br/>`) just before this which is not required. Can we please get rid of that?
The same is true for `asset-events-properties.md` as well.